source/Octopus.Tentacle.Contracts/ScriptServiceV3Alpha/StartScriptCommandV3Alpha.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace Octopus.Tentacle.Contracts.ScriptServiceV3Alpha
+{
+    public class StartScriptCommandV3Alpha : ScriptCommandV3Alpha
```

Comment:
Just thinking out aloud, how much do we want to force customers to specify the `scriptExecutionContext`?

To start with the K8 version will be very rarely used. 

Even when it takes off any everyone thinks it's amazing, I would guess the local shell context is more commonly used.

This makes me wonder, should we make it easier to create these types of commands somehow?

Either by 
- making a constructor that doesn't take the context, and then default it to the local shell context
- some factory style methods?

We don't have to do any of this, just some thoughts 🙂

Reply:
I believe in the server codebase we have some builders that are used for this purpose. In this case, we would default the context to the local shell execution, and then have methods for setting the required k8s context information, which would then build it up.